datadreamer/pipelines/generate_dataset_from_scratch.py

```diff
@@ -54,6 +58,8 @@
 
 det_annotators = {"owlv2": OWLv2Annotator}
 clf_annotators = {"clip": CLIPAnnotator}
+inst_seg_annotators = {"owlv2-slimsam": SlimSAMAnnotator}
+inst_seg_to_det = {"owlv2-slimsam": OWLv2Annotator}
```

Comment:
Isn't the dict name confusing?

Reply:
Is `inst_seg_detectors` better, @sokovninn?

Reply:
Yep, makes more sense to me